en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+# Auto-tuning
+
+Auto-tuning automates the process of tuning PX4 rate and attitude controllers.
+These are the most important controllers for stable and responsive flight (other tuning is more "optional").
+
+Tuning only needs to be done once, and is recommended unless unless you're using vehicle that has already been tuned by the manufacturer (and not modified since).
+
+:::warning
+Auto-tuning is performed while flying.
+The airframe must fly well enough handle moderate disturbances, and should be closely attended:
+- Test that your vehicle is [stable enough for autotuning](#pre-tuning-test).
+- Be ready to abort the autotuning process by moving remote control sticks.
+- Verify that the vehicle flies well after tuning.
+:::
+
+
+## Overview
+
+PX4 uses [PID controllers](../flight_stack/controller_diagrams.md) (rate, attitude, velocity, and position) to calculate the outputs required to move a vehicle from its current estimated state to match a desired setpoint.
+The controllers must be well tuned in order to get the best performance out of a vehicle.
+In particular, a poorly tuned rate controller results in less stable flight in all modes, and takes longer to recover from disturbances.
+
+Generally if you use an [airframe configuration](../config/airframe.md) that is similar to your vehicle then the vehicle will be able to fly.
+However unless the configuration precisely matches your hardware you should tune the rate and attitude controllers.
+Tuning the velocity and position controllers is less important because they are less affected by vehicle dynamics, and the default tuning configuration for a similar airframe is often sufficient.
+
+Autotuning provides an automatic mechanism to tune the rate and attitude controllers.
+It can be used to tune fixed wing and multicopter vehicles, and VTOL vehicles when flying as a multicopter or as a fixed wing (transition between modes must be manually tuned).
+In theory it should work for other vehicle types that have a rate controller, but currently only the above types are supported.
+
+Automatic tuning works well for the multicopter and fixed wing vehicle configurations supported by PX4, provided the frame is not too flexible
+(see [below for more information](#does-autotuning-work-for-all-supported-airframes)).
+
+The vehicle must be flying in [Hold mode](../flight_modes/hold.md), [Position mode](../flight_modes/position_mc.md), or [altitude mode](../flight_modes/altitude_mc.md).
```

Comment:
Is this true? I.e. can I tune in "hold mode"? Does it matter. The original docs appear to indicate that you can't assume that position hold mode will work and are written that way. Should I do so too?

Reply:
You can and I would recommend tuning in hold or position mode if the drone can already fly in those modes with the default tuning (which should be the case if it passes the `#pre-tuning-test`). In the original doc, we wanted to avoid having a user performing an automatic takeoff without knowing if the drone was able to fly.